app/assets/stylesheets/common/base/modal.scss

```diff
@@ -201,9 +201,35 @@
     }
 
     &.insert-link {
+      overflow-y: visible;
       input {
         min-width: 300px;
       }
+
+      .inputs {
```

Comment:
did you test this on mobile ?

Reply:
Yes, I did, the overflow doesn't apply on mobile. Here's a screenshot: 

![IMG_70DF6904735D-1](https://user-images.githubusercontent.com/368961/66596958-e7882c80-eb6b-11e9-9849-3693611b6aa7.jpeg)